hazelcast/src/test/java/com/hazelcast/spi/impl/operationservice/impl/Invocation_BlockingTest.java

```diff
@@ -276,7 +276,7 @@ private void assertFailsEventuallyWithOperationTimeoutException(final BiConsumer
 
     /**
      * Tests if the future on a blocking operation can be shared by multiple threads. This tests fails in 3.6 because
-     * only 1 thread will be able to swap out CONTINUE_WAIT and all other threads will fail with an OperationTimeoutExcepyion
+     * only 1 thread will be able to swap out CONTINUE_WAIT and all other threads will fail with an OperationTimeoutException
```

Comment:
```suggestion
     * only 1 thread will be able to swap out CONTINUE_WAIT and all other threads will fail with an OperationTimeoutException
```
```suggestion
     * only 1 thread will be able to swap out CONTINUE_WAIT and all other threads will fail with an {@link com.hazelcast.core.OperationTimeoutException}
```

Reply:
> I'm assuming these two suggestions are identical (GitHub's suggestion/files view interfaces are not perfect and make this stuff moderately painful...)

I added the exception reference as a Javadoc link, so there’s a chance at compile time an IDE can spot an incorrect reference.